### ISSUE TITLE
The Breadcrumb display is incomplete

### ISSUE THREAD

**Issue by lwdillon** [mkpaz/atlantafx#69]
When Breadcrumbs have a long item name or a large number of items, the display is incomplete, as shown in the following figure
![00dd3b93084de7f52f78a0b490c8cb2c](https://github.com/mkpaz/atlantafx/assets/14171008/1fe66dc5-c499-404d-9004-2952141d1072)

**Comment by mkpaz** [comment]
Yes, the breadcrumbs do not support auto hiding items when their size exceeds the container width. It's on the backlog, but for now you can only limit the number of items to ensure they fit within the available width. I implemented this in the FileManager [showcase](https://github.com/mkpaz/atlantafx/blob/master/sampler/src/main/java/atlantafx/sampler/page/showcase/filemanager/FileManagerPage.java#L188).
